docs/MVS_parameters_list.csv

```diff
@@ -47,4 +47,4 @@
 45, None, Length of the time-steps.,60, None, Numeric, Minutes,timestep,timestep-label
 46, None, The type of the component., demand, *demand*, str, None,type_asset,typeasset-label
 47, None," Input the type of OEMOF component. For example, a PV plant would be a source, a solar inverter would be a transformer, etc.  The `type_oemof` will later on be determined through the EPA.", sink, *sink* or *source* or one of the other component classes of OEMOF., str, None,type_oemof,typeoemof-label
-48, None, Unit associated with the capacity of the component.," Storage could have units like kW or kWh, transformer station could have kVA, and so on.", Appropriate scientific unit, str, NA,unit,unit-label
+48, None, Unit associated with the capacity of the component.," Storage could have units like kW or kWh, transformer station could have kVA, and so on.", Appropriate scientific unit, str, NA,unit,unit-label
```

Comment:
Please also add `scenario_description` to this table.

Reply:
The parameters seem to have been added in an alphabetical order her. scenario_description should go around number 35. Do I have to manually correct all the following numbers? @smartie2076